### PR TITLE
MSSQL fixes

### PR DESCRIPTION
**Pull request by ryanemerson**

### CODE REVIEW

File: src/main/java/io/gingersnapproject/k8s/configuration/DatabaseServiceBindingConverter.java [hunk 1]
```diff
@@ -52,6 +52,9 @@ public Optional<ServiceBindingConfigSource> convert(List<ServiceBinding> allBind
             .stream()
             .collect(Collectors.toMap(e -> Database.property(e.getKey()), Map.Entry::getValue));
       properties.put(Database.property("type"), type);
+      if (log.isDebugEnabled()) {
+         properties.forEach((k, v) -> log.debugf("%s=%s", k, v));
```
Comment (jabolina) [inline src/main/java/io/gingersnapproject/k8s/configuration/DatabaseServiceBindingConverter.java:56]: Some sort of sensitive information here?

**Review by jabolina — APPROVED** [review]
Everything is working fine locally. I'll look into adding MS to the tests. We can run it with ISPN until we have a cache-manager image.

File: deploy/mssql/mssql-compose.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ services:
     environment:
       ACCEPT_EULA: "Y"
       SA_PASSWORD: Password!42
+      MSSQL_AGENT_ENABLED: "true"
```
[the remark this answers is not in the document]
Comment (ryanemerson) [reply deploy/mssql/mssql-compose.yaml:10]: I read a LOT of MS docs about the agent before I stumbled across a page detailing the available env vars in the container, they do not promote this well!

File: src/main/java/io/gingersnapproject/k8s/configuration/DatabaseServiceBindingConverter.java (hunk 1 above)
Comment (ryanemerson) [reply src/main/java/io/gingersnapproject/k8s/configuration/DatabaseServiceBindingConverter.java:56]: Yes the username and password will be output, however we'll never release the Operator with DEBUG logs enabled. So I don't think this is a big concern. Having these logs available for debugging locally is invaluable though

**Comment by jabolina** [comment]
Merged, super thanks for the improvements @ryanemerson

**Pull request merged**